test/events.test.tsx

```diff
@@ -118,11 +118,70 @@ describe('Events', () => {
     await userEvent.pointer({ target: getCellsAtRowIndex(0)[1], keys: '[MouseRight]' });
     expect(getCellsAtRowIndex(0)[1]).toHaveAttribute('aria-selected', 'true');
   });
+
+  it('should call onSelectedCellChange when cell is clicked', async () => {
+    let idx: number;
+    let rowIdx: number;
+    let row: Row;
+    let column: CalculatedColumn<Row> | undefined;
+
+    render(
+      <EventTest
+        onSelectedCellChange={(args) => {
+          idx = args.idx;
+          rowIdx = args.rowIdx;
+          row = args.row;
+          column = args.column;
+        }}
+      />
+    );
+
+    const expectOnCellSelectedFired = () => {
+      expect(getCellsAtRowIndex(0)[1]).toHaveAttribute('aria-selected', 'true');
+      expect(idx).toBe(1);
+      expect(rowIdx).toBe(0);
+      expect(row).toStrictEqual({
+        col1: 1,
+        col2: 'a1'
+      });
+      expect(column?.key).toBe('col2');
+    };
+
+    // Selected by click
+    await userEvent.click(getCellsAtRowIndex(0)[1]);
+    expectOnCellSelectedFired();
+
+    // Selected by double click
+    await userEvent.dblClick(getCellsAtRowIndex(0)[1]);
+    expectOnCellSelectedFired();
+
+    // Selected by right-click
+    await userEvent.pointer({ target: getCellsAtRowIndex(0)[1], keys: '[MouseRight]' });
+    expectOnCellSelectedFired();
+
+    // Selected by ←↑→↓ keys
+    await userEvent.click(getCellsAtRowIndex(1)[1]);
+    await userEvent.keyboard('{ArrowUp}');
+    expectOnCellSelectedFired();
+
+    // Selected by tab key
+    await userEvent.click(getCellsAtRowIndex(0)[0]);
+    await userEvent.keyboard('{Tab}');
+  });
+
+  it('should call onSelectedCellChange only once when cell is double clicked', async () => {
+    let timesToBeCalled = 0;
+
+    render(<EventTest onSelectedCellChange={() => timesToBeCalled++} />);
+    await userEvent.dblClick(getCellsAtRowIndex(0)[0]);
+
+    expect(timesToBeCalled).toBe(1);
```

Comment:
```suggestion
    expect(onSelectedCellChangeSpy).toHaveBeenCalledTimes(1);
```

Reply:
I updated the test, but I'm not familiar with spying, so can you check if I have implemented as you expected?